connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -3594,6 +3543,72 @@ public void testHerderStopServicesClosesUponShutdown() {
         assertEquals(0, shutdownCalled.getCount());
     }
 
+    @Test
+    public void testPollDurationOnSlowConnectorOperations() {
+        connectProtocolVersion = CONNECT_PROTOCOL_V1;
+        // If an operation during tick() takes some amount of time, that time should count against the rebalance delay
+        int rebalanceDelayMs = 20000;
+        long operationDelayMs = 10000;
+        long maxPollWaitMs = rebalanceDelayMs - operationDelayMs;
```

Comment:
It helps clarify intent with these cases if these constants declared at the beginning are marked `final`:
```suggestion
        final int rebalanceDelayMs = 20000;
        final long operationDelayMs = 10000;
        final long maxPollWaitMs = rebalanceDelayMs - operationDelayMs;
```